miniapps/gslib/findpts.cpp

```diff
@@ -36,6 +36,11 @@
 //    findpts -m ../../data/inline-wedge.mesh -o 3
 //    findpts -m ../../data/amr-quad.mesh -o 2
 //    findpts -m ../../data/rt-2d-q3.mesh -o 3 -mo 4 -ft 2
+//    findpts -m ../../data/square-mixed.mesh -o 2 -mo 2
```

Comment:
I think there should be an example with the `-hr` and `-pr` options. Note that `-hr` and `-pr` do not work for pyramids, so there should probably also be a check for that.